tests/integration-tests/tests/createami/test_createami.py

```diff
@@ -480,5 +481,12 @@ def _test_build_image_failed(image):
 
     if image.image_status == "BUILD_FAILED":
         image.keep_logs = True
-
+        _keep_last_hundred_lines_log(image)
     assert_that(image.image_status).is_equal_to("BUILD_FAILED")
+
+
+def _keep_last_hundred_lines_log(image):
```

Comment:
Perhaps we can make this function less coupled to the length of the logs i.e.
```
def _tail_logs(image, limit=100):
    ...
    failure_logs = image.get_log_events(log_stream_name, start_from_head=True, query="events[*].message", limit=limit)
    ...
```
That way we won't need to change the function name or body if we need to increase/reduce the number of logs.

Reply:
I don't think it is necessary to expose that parameter. I expected we would like to have same numbers of log for all the build_image integration tests.

Reply:
Noted. Let's make the function name agnostic of the number of lines then e.g. `_keep_recent_logs`.
This is to avoid function name changes if we ever need to update the number of lines being logged.